metecho/api/models.py

```diff
@@ -400,6 +402,24 @@ def finalize_populate_github_users(self, *, error=None, originating_user_id):
         else:
             self.notify_error(error, originating_user_id=originating_user_id)
 
+    def queue_refresh_github_issues(self, *, originating_user_id):
+        from .jobs import refresh_github_issues_job
+
+        if not self.currently_fetching_issues:
+            self.currently_fetching_issues = True
+            self.save()
```

Comment:
I think we also want a `self.notify_changed(originating_user_id=originating_user_id)` after this `save()`?

Reply:
Fixed in 94e344587db6a311467b76c6be3f1831c8a2c96f